### DISCUSSION TITLE
New v0.5.0 release?

### DISCUSSION THREAD

**Discussion by sodul** [uber-go/mock#188] [General]
The v0.4.0 release was done in December 2023 and there has been 14 commits to main since, including fixes for regressions in behavior introduced in v0.3.0.

**Comment by sodul** [comment]
@r-hang @sywhang Any input on getting newer releases?

I'm a tad concern that the large gap between official release will allow for regressions to be bundled with fixes that were done for v0.4.0. It is good to see that the main branch is being maintained but we are stuck on v0.2.0 due to regressions in behavior in v0.3.0 which were fixed in main after the v0.4.0 release.

**Comment by sywhang** [comment]
Thanks for bringing this up. We'll track the new release for this month.

**Comment by sodul** [comment]
@sywhang do you have an updated ETA?
